http/http-producer/pom.xml

```diff
@@ -32,12 +32,8 @@
             <version>${log4j.version}</version>
         </dependency>
         <dependency>
-            <groupId>io.jaegertracing</groupId>
-            <artifactId>jaeger-client</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.opentracing.contrib</groupId>
-            <artifactId>opentracing-kafka-client</artifactId>
```

Comment:
What does removing of the OpenTracing stuff mean for backwards compatibility and testing the OpenTracing stuff?

Reply:
All tracing (and this includes OpenTracing) moved to `tracing` module.
And it's all back compatible.